manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -545,3 +539,80 @@ where
         )
     }
 }
+
+impl<C, CV> PointAdd<Compiler<C>> for GroupVar<C, CV>
+where
+    C: ProjectiveCurve,
+    CV: CurveVar<C, ConstraintField<C>>,
+{
+    type Output = Self;
+
+    fn add(&self, rhs: &Self, compiler: &mut Compiler<C>) -> Self::Output {
```

Comment:
Please `#[inline]`. I'd like to inline all `impl Trait` methods for now.

Reply:
done